app/global.R

```diff
@@ -30,9 +30,9 @@ HOSPITALIZATIONS_AHEAD_OPTIONS <- c(
 )
 
 # Earliest 'as of' date available from covidcast API
-MIN_AVAIL_NATION_AS_OF_DATE <- as.Date("2021-01-09")
-MIN_AVAIL_HOSP_AS_OF_DATE <- as.Date("2020-11-11")
-MIN_AVAIL_TERRITORY_AS_OF_DATE <- as.Date("2021-02-10")
+MIN_AVAIL_NATION_AS_OF_DATE <- as.Date("2020-04-02")
+MIN_AVAIL_HOSP_AS_OF_DATE <- as.Date("2020-11-16")
```

Comment:
Not sure if this matters yet but flu signals from hhs are only available back to as_of 2021-09-20

Reply:
:+1: will keep an eye on that.